src/common/IOSLauncher.ts

```diff
@@ -66,13 +79,6 @@ export class IOSLauncher {
                 );
                 return IOSUtils.bootDevice(deviceUDID);
             })
-            .then(() => {
-                CommonUtils.startCliAction(
-                    `Launching`,
-                    `Waiting for device ${deviceUDID} to boot`
-                );
-                return IOSUtils.waitUntilDeviceIsReady(deviceUDID);
-            })
```

Comment:
Similar to `AndroidUtils`, the function `IOSUtils..bootDevice() `now takes a param (defaulting to true) indicating whether it should wait for the simulator and will internally call `waitUntilDeviceIsReady` so no need to do this here anymore.